testing/terraform/APKBUILD

```diff
@@ -2,37 +2,36 @@
 # Contributor: Gennady Feldman <gena01@gmail.com>
 # Maintainer: Thomas Boerger <thomas@webhippie.de>
 pkgname=terraform
-pkgver=0.7.12
+pkgver=0.8.1
 pkgrel=0
 pkgdesc="Building, changing, and combining infrastructure safely and efficiently"
 url="https://www.terraform.io/"
-arch="x86_64"
+arch="all"
 license="MPL 2.0"
 depends=""
-makedepends="go"
+depends_dev=""
```

Comment:
As @clandmeter said on IRC, having `$depends_dev` is useless if there's no `$subpackages`.